setup.py

```diff
@@ -0,0 +1,68 @@
+import os
+import platform
+import re
+from setuptools.command.sdist import sdist
+from setuptools import setup, Extension, find_packages
+from setuptools.command.build_ext import build_ext
+from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
+import subprocess
+import tarfile
+
+with open("README.md", "r") as f:
+    readme = f.read()
+
+def get_version():
+    path = './dist/fasttrackml_linux_x86_64' if platform.system() == "Linux" else './dist/fasttrackml_macos_x86_64'
+    bin = tarfile.open(f'{path}.tar.gz')
+    bin.extractall(path)
+    bin.close()
+    version_string = subprocess.check_output(
+        [f"{path}/fml", "--version"], universal_newlines=True
+    ).strip()
+    matches = re.findall(r'\d+\.\d+\.\d+', version_string)
+    return matches[0]
```

Comment:
Use the `VERSION` environment variable (with a fallback to `'dev'`)

Reply:
I believe 'dev' is not a valid version for setuptools

Reply:
you're right!